sdk/include/ndk/kefuncs.h

```diff
@@ -33,6 +33,54 @@ extern "C"
 
 #ifndef NTOS_MODE_USER
 
+//
+// Affinity helper
+//
+FORCEINLINE KAFFINITY AFFINITY_MASK(ULONG Index)
+{
+    ASSERT(Index < sizeof(KAFFINITY) * 8);
+    return (KAFFINITY)1 << Index;
+}
+#ifdef _WIN64
```

Comment:
It may be preferable (imho) to just do the ifdef only within the implementation of the inlined function. Like this for example:
```c
FORCEINLINE BOOLEAN BitScanForwardAffinity(PULONG Index, KAFFINITY Mask)
{
#ifdef _WIN64
    return BitScanForward64(Index, Mask);
#else
    return BitScanForward(Index, Mask);
#endif
}

Reply:
done